tests/addon/test_integration.py

```diff
@@ -6559,3 +6562,118 @@ def test_reschedules_are_ignored(
                 if revlog_type != REVLOG_RESCHED
             ]
             assert dates_from_summaries == expected_dates
+
+
+@pytest.mark.sequential
+def test_terms_agreement_not_accepted_with_reviewer_sidebar_instance(
+    anki_session_with_addon_data: AnkiSession,
+    qtbot: QtBot,
+    install_ah_deck: InstallAHDeck,
+    import_ah_note: ImportAHNote,
+    mocker: MockerFixture,
+    requests_mock: requests_mock.Mocker,
+):
+    entry_point.run()
+    message = TERMS_AGREEMENT_NOT_ACCEPTED
+    requests_mock.get("https://app.ankihub.net/api/users/me", json={"is_premium": True})
+    with anki_session_with_addon_data.profile_loaded():
+        ah_did = uuid.uuid4()
+        install_ah_deck(ah_did=ah_did)
+
+        # Changes the deck setting so that there are unsuspend cards ready for review
+        config.set_suspend_new_cards_of_new_notes(ankihub_did=ah_did, suspend=False)
+        deck_config = config.deck_config(ah_did)
+        import_ah_note(
+            ah_did=ah_did,
+            anki_did=deck_config.anki_id,
+        )
+        aqt.mw.col.decks.set_current(deck_config.anki_id)
```

Comment:
:pick: The default value for `suspend_new_cards_of_new_notes` is `False`, so this works as well.
```suggestion
        anki_did = 1
        ah_did = install_ah_deck(anki_did=anki_did)
        import_ah_note(ah_did=ah_did, anki_did=anki_did)

        aqt.mw.col.decks.set_current(anki_did)
```